### PR TITLE
Added a background and a path

### PR DESCRIPTION
**Pull request by lawin666**
I made a repeated background because we need the background to start with and I think in the future we will need some levels and we can begin with this map as level one. We don't need a complex background in the first level; a repeated background is a simple map that will regenerate the same thing repeatedly.

I used a picture resembling grass and a picture of stones as a path. I wanted the background to have a dark theme so it could fit the game theme. For the dark green picture that looks like dark green grass, I scaled it to three to make it bigger and in ParallaxLayer inspectors I changed mirroring to x-3072 and y-3072 because it was going to glitch when the player was moving to the right. The background wasn't repeated in time so I needed to put mirroring to x-3072 and y-3072 so it could fit properly with fullscreen mode.

### CODE REVIEW

**Comment by Snurrigt1** [comment]
Change camera zoom to 0.8

File: Player/player.tscn
Comment (xix-xeaon) [inline Player/player.tscn:1]: Why are there changes to the player?

**Review by xix-xeaon — CHANGES_REQUESTED** [review]
I believe there are some unnesseccary changes that probably shouldn't be included in this PR. But other than that I think this is acceptable. In a future PR someone should probably improve this to use tilemaps or similar to make it easier to work with.

**Comment by xix-xeaon** [comment]
I see what your intention is now. There are a few things I'm not happy with but I suppose we'll have to merge soon anyway and fix those things later.

However, the unrelated changes to the project file are still blocking this PR from being merged.

**Review by xix-xeaon — APPROVED** [review]

**Pull request merged**